raiden/transfer/channel.py

```diff
@@ -1721,10 +1852,19 @@ def handle_receive_withdraw_request(
         channel_state=channel_state, withdraw_request=withdraw_request
     )
     if is_valid:
+        withdraw_state = WithdrawState(
+            total_withdraw=withdraw_request.total_withdraw,
+            nonce=withdraw_request.nonce,
+            # Expiration for a received request is set to 0,
+            # as only the sender can instruct the current node
+            # to expire the withdraw.
+            expiration=BlockExpiration(0),
```

Comment:
I don't understand why this value is being reset, how can you validate a `WithdrawRemovedExpired` if you don't know when it expires?

Reply:
This was a mistake. Fixed